src/cmd.js

```diff
@@ -112,7 +107,7 @@ function _cmd(options, command, commandArgs, userOptions) {
   } else {
     stdout = result.stdout.toString();
     stderr = result.stderr.toString();
-    code = result.code;
+    code = result.exitCode || result.errno || 0;
```

Comment:
I see exitCode in the docs, but `errno` is only mentioned as part of the exception object. Do we really need this in execa v5?

https://github.com/sindresorhus/execa/tree/v5.0.0?tab=readme-ov-file#exitcode

Reply:
you're right, i'll remove it.